node/runtime/src/lib.rs

```diff
@@ -932,9 +955,27 @@ static EVM_CONFIG: EvmConfig = EvmConfig {
 	has_ext_code_hash: true,
 	estimate: false,
 };
+/// Current (safe) approximation of the gas/s consumption considering
+/// EVM execution over compiled WASM.
+pub const GAS_PER_SECOND: u64 = 8_000_000;
+
+/// Approximate ratio of the amount of Weight per Gas.
+/// u64 works for approximations because Weight is a very small unit compared to gas.
+pub const WEIGHT_PER_GAS: u64 = WEIGHT_PER_SECOND / GAS_PER_SECOND;
+
+pub struct EdgewareGasWeightMapping;
+
+impl pallet_evm::GasWeightMapping for EdgewareGasWeightMapping {
+	fn gas_to_weight(gas: u64) -> Weight {
+		Weight::try_from(gas.saturating_mul(WEIGHT_PER_GAS)).unwrap_or(Weight::MAX)
+	}
+	fn weight_to_gas(weight: Weight) -> u64 {
+		weight.wrapping_div(WEIGHT_PER_GAS)
+	}
+}
 
 impl pallet_evm::Config for Runtime {
-	type FeeCalculator = FixedGasPrice;
+	type FeeCalculator = (); // TODO: permit validators to set minimum gas prices
```

Comment:
I think we should still use the FixedGasPrice as this is at the runtime level. The validators setting it is at the client level, no?

Reply:
All this means is that there's no minimum gas price at the runtime level -- we could just add a struct that says "0" if we wanted to be explicit about it.